### PR TITLE
Add container image building capabilities

### PR DESCRIPTION
**Pull request by lnovara**
As per PR title.

### CODE REVIEW

File: modules/monitoring/custom/grafana/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# From: https://github.com/grafana/grafana/blob/81957732abf01c9df88307c5f41a12c9f117c0c2/packaging/docker/custom/Dockerfile
+
+ARG GRAFANA_VERSION="latest"
```
Comment (nutellinoit) [inline modules/monitoring/custom/grafana/Dockerfile:3]: I would use a generic version ARG like IMAGETAG to use the same array we have for the tags, this way we can sync and build multiple versions with the same modifications

File: modules/monitoring/images.yml [hunk 2]
```diff
@@ -52,15 +52,15 @@ images:
...
-      - "7.3.6"
-      - "7.5.7"
-      - "8.0.6"
-      - "8.1.2"
-      - "8.3.3"
+    source: grafana
+    build:
+      context: custom/grafana
+      args:
+        - name: GRAFANA_VERSION
```
Comment (nutellinoit) [inline modules/monitoring/images.yml:59]: I would drop this arg in favor of the tag: array, see the comment on the `Dockerfile`

**Review by nutellinoit — CHANGES_REQUESTED** [review]
Some comments on the main tag for the build. 

Is it working locally?

File: modules/monitoring/custom/grafana/Dockerfile (hunk 1 above)
Comment (lnovara) [reply modules/monitoring/custom/grafana/Dockerfile]: I used the original Dockerfile from Grafana, but your suggestion makes a lot of sense.

File: modules/monitoring/images.yml (hunk 2 above)
Comment (lnovara) [reply modules/monitoring/images.yml]: Done. ✅

**Review by nutellinoit — APPROVED** [review]
LGTM!

**Pull request merged**